modules/installation-about-mirror-registry.adoc

```diff
@@ -7,18 +7,19 @@
 [id="installation-about-mirror-registry_{context}"]
 = About the mirror registry
 
-You can mirror the images that are required for {product-title} installation and subsequent product updates to a mirror registry. These actions use the same process. The release image, which contains the description of the content, and the images it references are all mirrored. In addition, the Operator catalog source image and the images that it references must be mirrored for each Operator that you use. After you mirror the content, you configure each cluster to retrieve this content from your mirror registry.
+You can mirror the images that are required for {product-title} installation and subsequent product updates to a container mirror registry such as Red Hat Quay, JFrog Artifactory, Sonatype Nexus Repository, or Harbor. If you do not have access to a large-scale container registry, you can use the _mirror registry for Red Hat Openshift_, a small-scale container registry included with all {product-title} subscriptions.
```

Comment:
remove the use of  `all`, and just say `_mirror registry for Red Hat Openshift_, a small-scale container registry included with {product-title} subscriptions.`